docs/eternalcore/notifications.md

```diff
@@ -2,78 +2,138 @@
+```
 
-::: code-group
+:::
+
+### Examples combining multiple notifications:
+👑 Unleash your creativity and design your own custom notifications without any limits!
 
-```yaml [CHAT + ACTIONBAR]
+#### Chat + Actionbar
+```yaml
  # Displays a message in the chat and action bar.
- example: "[CHAT, ACTIONBAR] Hello world!"
+ example:
+    chat: "Hello world!"
+    actionbar: "Hello world!"
 ```
 
-```yaml [TITLE + SUBTITLE]
+#### Title + Subtitle
+```yaml 
 # Displays a message in the title and subtitle.
-example: "[TITLE, SUBTITLE] Hello world!" 
+example: 
+    title: "Hello world!"
+    subtitle: "Hello world!"
 ```
 
-```yaml [CHAT + ACTIONBAR + TITLE + SUBTITLE]
+#### Title + Subtitle with specified times
+```yaml
+# Displays a message in the title and subtitle.
+example: 
+    title: "Hello world!"
+    subtitle: "Hello world!"
+    times: "1s 2s 1s" # <- The first number is the time it takes to fade in, the second is the time it takes to stay on the screen, and the third is the time it takes to fade out.
+```
+
+#### Chat + Actionbar + Title + Subtitle
+```yaml 
 # Displays a message in the chat, action bar, title, and subtitle.
-example: "[CHAT, ACTIONBAR, TITLE, SUBTITLE] Hello world!" 
+example: 
+    chat: "Hello world!"
+    actionbar: "Hello world!"
+    title: "Hello world!"
+    subtitle: "Hello world!"
 ```
 
-```yaml [DISABLED]
-# Does not display a message.
-example: "[DISABLED]" 
+#### Chat + Actionbar + Title + Subtitle with specified times
+```yaml 
+# Displays a message in the chat, action bar, title, and subtitle with specified times for title and subtitle
+example: 
+    chat: "Hello world!"
+    actionbar: "Hello world!"
+    title: "Hello world!"
+    subtitle: "Hello world!"
+    times: "1s 2s 1s" # <- The first number is the time it takes to fade in, the second is the time it takes to stay on the screen, and the third is the time it takes to fade out.
 ```
 
-:::
+#### Chat + Actionbar + Title + Subtitle + Sound
+```yaml 
+# Displays a message in the chat, action bar, title, and subtitle with specified times for title and subtitle
+example: 
+    chat: "Hello world!"
+    actionbar: "Hello world!"
+    title: "Hello world!"
+    subtitle: "Hello world!"
+    sound: "ENTITY_PLAYER_LEVELUP 2.0 1.0" # <- The first number is the volume, the second is the pitch.
+```
+
+#### Chat + Actionbar + Title + Subtitle + Sound + Category
+```yaml 
+# Displays a message in the chat, action bar, title, and subtitle with specified times for title and subtitle
+example: 
+    chat: "Hello world!"
+    actionbar: "Hello world!"
+    title: "Hello world!"
+    subtitle: "Hello world!"
+    sound: "ENTITY_PLAYER_LEVELUP WEATHER 2.0 1.0" # <- The first number is the volume, the second is the pitch.
+    # If you want to play a sound in a certain category, for example if a player has the sound category "WEATHER" in the game settings set to 0%, the sound will not play.
```

Comment:
WARNING: Players volume settings can mute certain categories of sounds.

Reply:
Defaultowo jest master, a jak ktoś chcę przypisać dźwięk do jakiejś kategorii to ma taką możliwość